### DISCUSSION TITLE
Set cookies in server actions

### DISCUSSION THREAD

**Discussion by artemavrin** [vercel/next.js#49375] [Help] [answered]
### Summary

According this [docs](https://nextjs.org/docs/app/api-reference/functions/cookies#cookiessetname-value-options) we can use set() method in server actions, but we can not.
I have an error Property 'set' does not exist on type 'ReadonlyRequestCookies'
bug?

**Comment by aXenDeveloper** [comment, marked as answer]
Hi,
There is open issue => https://github.com/vercel/next.js/issues/49259
